src/components/sidebar-area/sidebar-area.tsx

```diff
@@ -114,25 +123,33 @@ export function SidebarArea() {
                 }}
               >
                 <StatusCircle isConnected={isConnected} isLoading={isLoading} />
-                {hostname}
+                {isLoading ? (
+                  <LoaderCircle className='animate-spin' size={12} />
+                ) : (
+                  hostname
+                )}
               </div>
               <TooltipContent>
-                {process.env.NEXT_PUBLIC_LETTA_SERVER_URL || 'http://localhost:8283'}
+                {isLoading ? (
+                  <LoaderCircle className='animate-spin' size={12} />
```

Comment:
We should just not show the tooltip if our content is loading, or instead write "Determining hostname" in it's place, instead of hovering on a spinner and getting another spinner